contrib/completions/zsh/kubectl

```diff
@@ -0,0 +1,214 @@
+#!/bin/zsh
```

Comment:
This file was just created my hand right? Not generated in any way? Should it be a separate PR?